queries.sql

```diff
@@ -23,3 +23,52 @@ SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 
 /* Find all animals with a weight between 10.4kg and 17.3kg */
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/*  update the animals table by setting the species column to unspecified inside transaction and roll back */
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+/* Update species */
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species = '';
+COMMIT;
```

Comment:
- [ ]  For a better query, I suggest modifying this query from `UPDATE animals SET species = 'pokemon' WHERE species = '';` to `UPDATE animals SET species = 'pokemon' WHERE species IS NULL;`. This is to make use of `NULL` instead of the empty string ''.

- [ ] Kindly verify that change was made after updating the animals table before committing.